test/license_test.rb

```diff
@@ -66,7 +66,7 @@
     end
 
     it "returns true if the normalized content is the same" do
-      license = Licensed::License.new({}, "test content")
+      license = Licensed::License.new({}, "- test content")
```

Comment:
normalization rules changed in `licensee` that caused this to fail.  a leading `*` is now normalized as a list indicator rather than general markup, so I've updated the test to check that it's normalized to another list indicator `-`